Sources/Controllers/2.0/ConfigureScreen/OAConfigureMenuMainScreen.mm

```diff
@@ -87,7 +88,7 @@ - (void) setupViewInternal
                                             @"key" : @"quick_action",
                                             @"img" : @"ic_custom_quick_action",
                                             @"selected" : @([_settings.quickActionIsOn get]),
-                                            @"color" : [_settings.quickActionIsOn get] ? UIColorFromRGB(0xff8f00) : [NSNull null],
+                                            @"color" : UIColorFromRGB(_settings.applicationMode.getIconColor),
```

Comment:
Why do you store the color at all? Simply use ```_settings.applicationMode.getIconColor``` in the cellForRow method. This implementation will probably cause a bug if you switch to another profile.